### DISCUSSION TITLE
new member

### DISCUSSION THREAD

**Discussion by umermalik053** [community/community#52668] [New to GitHub]
hello my name is umar malik i am a web developer i am newly join this community

**Comment by neutrinomuon** [comment]
Hello, @umermalik053, welcome to GitHub. I'm confident that we will all learn together (including you) as a community.

**Comment by queenofcorgis** [comment]
Welcome to GitHub and the GitHub Community @umermalik053  🎆 ! We're glad you're here and can't wait to see where your GitHub journey takes you. Check out this [check-in](https://github.com/orgs/community/discussions/42302#discussion-4683140) with links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist. 

Best of luck!
